body.tex

```diff
@@ -1,215 +1,336 @@
 
-Since the number of requirement for the Rubin Observatory is very high, the VCD is generated per subsystem.
+The VCD is a \LaTeX document generated using the document generation tool Docsteady.
+As for other documents the VCD is managed in a GitHub repository, built with Travis CI and published through LSST the Docs.
 
-The VCD provides 2 main sections:
+Since the number of requirement and verification elements for the Rubin Observatory is very high --
+approximately 700 requirements and 1000 verification elements in DM alone -- a separate VCD is generated for each subsystem.
+In the document there are 2 main sections:
 
 \begin{itemize}
-\item \textbf{Summary Information} where a status overview is given. 
-This includes the number of requirements and verification elements related to passed or failed test cases.
-\item \textbf{Detailed Information} where for each requirement it is shown which are the verification elements and test cases
-and the status of the test cases. Links to the test documents are provided, but not descriptive information.
+\item \textbf{Summary Information} where a status overview is given.
+This includes the number of requirements and verification elements covered by passed test cases.
+Figure \ref{fig:vcdsum} shows an example extracted from the DM VCD.
+\item \textbf{Detailed Information} where for each requirement, the verification elements, test cases
+and the status of the test cases are shown. Links to the test documents are provided, but not descriptive information.
+Figure \ref{fig:vcddetail} shows a detail example extracted from the DM VCD.
 \end{itemize}
 
-The VCD become an important document for management to know the level of verification and validation that has been achieved so far.
-At the same time it can be provided to reviews in order to demonstrate that the expected milestones have been met.
-
+The VCD is an important management tool assessing the  level of verification and validation that has been achieved so far.
+ It is also useful to provided to review panels to show which requirements have been verified, 
+ including for each of them a link to the test document were more detailed information can be found.
+% The test reports more readily show a milestone has been met ..
+
+\begin{figure}
+\begin{center}
+\includegraphics[width=\textwidth]{imgs/VCDsumm.png}
+ \caption{Example summary extracted from the DM VCD.}
+ \label{fig:vcdsum}
+\end{center}
+\end{figure}
+
+\begin{figure}
+\begin{center}
+\includegraphics[width=\textwidth]{imgs/VCDdetail.png}
+ \caption{Example of a detailed information for a requirement extracted from the DM VCD 
+ showing a Verification Element and 3 test cases, which 2 have been run and passed.}
+ \label{fig:vcddetail}
+\end{center}
+\end{figure}
+%might be good to put more description .. showing a Verlifcation element and 3 test cases 2 whihc have been run and passed.
```

Comment:
In this figure LSE-68 is mentioned but but is not explained or referenced  anywhere. It should be explained

Reply:
OK, I expand the caption.